src/Components/Misc/ExportFileButton.tsx

```diff
@@ -25,26 +41,38 @@ const labsToString = (lab: any) => {
 }
 
 export const ExportFileButton = () => {
-  const fileExportRef = React.useRef<HTMLAnchorElement | null>(null);
+  const fileExportPeopleRef = React.useRef<HTMLAnchorElement | null>(null);
+  const fileExportCoursesRef = React.useRef<HTMLAnchorElement | null>(null);
 
   const handleClick = () => {
-    if (fileExportRef.current !== null) {
-      axios.get('https://y7nswk9jq5.execute-api.us-east-1.amazonaws.com/prod/export-schedule').then(response => response.data as AssignedPerson[])
+    if (fileExportPeopleRef.current !== null && fileExportCoursesRef.current !== null) {
+      axios.get('https://y7nswk9jq5.execute-api.us-east-1.amazonaws.com/prod/export-schedule').then(response => response.data as Response)
       .then(data => {
-        let csvText = 'First;Last;Classes;Labs;# LH\n'
-        for (let person of data) {
-          csvText += `${person.first};${person.last};${person.classes.join(', ')};${labsToString(person.labs)};${person.number_lab_hours}\n`
+        let coursesTsvText = 'Course ID\tSec\tLab Days\tBegin\tEnd\tInstructor\tLab Room\n'
+        for (let course of data.courses) {
+          coursesTsvText += `${course.course_id}\t${course.sec}\t${course.lab_days.join('')}\t${course.begin}\t${course.end}\t${course.instructor}\t${course.lab_room}\t${course.peer_teachers.join('\t')}\n`
         }
-        const href = `data:text/csv;charset=utf-8,${encodeURIComponent(csvText)}`;
-        fileExportRef.current?.setAttribute('href', href)
+        const courseshref = `data:text/tab-separated-values;charset=utf-8,${encodeURIComponent(coursesTsvText)}`;
+        fileExportCoursesRef.current?.setAttribute('href', courseshref)
+
+        let personTsvText = 'First\tLast\tClasses\tLabs\t# LH\n'
+        for (let person of data.people) {
+          personTsvText += `${person.first}\t${person.last}\t${person.classes.join(', ')}\t${labsToString(person.labs)}\t${person.number_lab_hours}\n`
+        }
+        const peoplehref = `data:text/tab-separated-values;charset=utf-8,${encodeURIComponent(personTsvText)}`;
+        fileExportPeopleRef.current?.setAttribute('href', peoplehref)
+      })
```

Comment:
This would best be refactored into API.ts where the response can be formatted and given directly to the export refs. I am also unable to test if it handles exporting sections with more than one person since it would require me to save another schedule, so I trust that you have tested it and it works well.